src/components/SearchMap.js

```diff
@@ -222,7 +230,10 @@ const styles = StyleSheet.create({
     bottomRowText: {
         color: 'white',
         fontWeight: 'bold',
-        alignSelf: 'center'
+        alignSelf: 'center',
+        textShadowColor: 'rgba(0, 0, 0, 0.75)',
+        textShadowOffset: {width: 1, height: 1},
+        textShadowRadius: 3
```

Comment:
Added a simple shadow to the site description text in the preview. We can keep tweaking this if you have any better ideas @amgoodman5